content/news/2025-09-24-bevy-0.17/index.md

```diff
@@ -451,38 +451,8 @@ expansion toggle).
 
 ### Widget Notifications
 
-Applications need a way to be notified when the user interacts with a widget. Currently in `bevy_ui_widgets`
-we're experimenting with a `Callback` system, which uses "one shot systems" under the hood. We're also heavily
-considering using Events / Observers for this. This is one of the primary reasons for the "experimental" label!
-
-### State Management
```

Comment:
Awwww...

My concern here is that novices are going to ask "why doesn't my slider 'just work' like it does on the web?" Why do I have to do the extra work of managing the state?